src/yb/consensus/log.cc

```diff
@@ -559,6 +559,12 @@ Status Log::Init() {
     RETURN_NOT_OK(reader_->GetSegmentsSnapshot(&segments));
     active_segment_sequence_number_ = segments.back()->header().sequence_number();
     LOG_WITH_PREFIX(INFO) << "Opened existing logs. Last segment is " << segments.back()->path();
+
+    if (metrics_) {
+      std::for_each(segments.begin(), segments.end(), 
+                    [this](auto segment) {
+                    this->metrics_->wal_size->IncrementBy(segment->file_size());});
+    }
```

Comment:
I believe this can be removed, since we're already incrementing when we close the last segment.

Reply:
@rahuldesirazu    This code would be called by the  `Log::Init()`  would initialize the `log_wal_size` metric ;  if remove it ,  the `log_wal_size` metric would  represents the number of bytes since server start,  instead of the number of bytes  already existing in the server  ?